partner_financial_risk/models/res_partner.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+# © 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+from openerp import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    risk_sale_order_include = fields.Boolean(
+        string='Include Sale Orders', help='Compute in total risk')
+    risk_sale_order_limit = fields.Monetary(
+        string='Limit Sale Orders', help='Set 0 if it not lock')
+    risk_sale_order = fields.Monetary(
+        compute='_compute_risk_sale_order', store=True,
+        string='Sale Order Not Invoiced')
+
+    risk_invoice_draft_include = fields.Boolean(
+        string='Include Draft Invoices', help='Compute in total risk')
+    risk_invoice_draft_limit = fields.Monetary(
+        string='Limit In Draft Invoices', help='Set 0 if it not lock')
+    risk_invoice_draft = fields.Monetary(
+        compute='_compute_risk_invoice', store=True,
+        string='Not Validated Invoice')
```

Comment:
Much better now. I know there are a lot of fields, but IMHO this module would benefit a lot from having `help` attributes that tell you what each field is used for.